### PR TITLE
[Translation] [russian] - translates embedding into Russian

### PR DESCRIPTION
**Pull request by koleaby4**

### CODE REVIEW

File: _content/tour/rus/embedding.article [hunk 1]
```diff
@@ -1,74 +1,74 @@
...
 
-- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Watch The Video]]
-- Need Financial Assistance, Use Our [[https://www.ardanlabs.com/scholarship/][Scholarship Form]]
+- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Посмотреть видео]]
+- Если вам нужна финансовая помощь - заполните нашу [[https://www.ardanlabs.com/scholarship/][заявку на стипендию]]
 
-Embedding types provide the final piece of sharing and reusing state and behavior
-between types. Through the use of inner type promotion, an inner type's fields and
-methods can be directly accessed by references of the outer type.
+Встраивание типов является финальным механизмом обмена и повторного использования состояния и поведения типов.
```
Comment (rumyantseva) [inline _content/tour/rus/embedding.article:9]: Two words for the dictionary: embedding and promotion. I like the translation for both, for promotion, I prefer "продвижение" just like it's chosen here, but I suggest mentioning "повышение" as a synonym as it's also used (e.g. in the book translation I often refer to).

File: _content/tour/rus/embedding.article [hunk 2]
```diff
@@ -1,74 +1,74 @@
...
+А вот это уже встраивание.
 
     type user struct {
         name  string
         email string
     }
 
     type admin struct {
-        user               // Value Semantic Embedding
+        user               // Встраивание семантики значения
```
Comment (rumyantseva) [inline _content/tour/rus/embedding.article:48]:
```suggestion
        user               // Встраивание c семантикой значения
```

File: _content/tour/rus/embedding.article [hunk 3]
```diff
@@ -1,74 +1,74 @@
...
+используя семантику указателя.
 
     type user struct {
         name  string
         email string
     }
 
     type admin struct {
-        *user              // Pointer Semantic Embedding
+        *user              // Встраивание семантики указателя
```
Comment (rumyantseva) [inline _content/tour/rus/embedding.article:61]:
```suggestion
        *user              // Встраивание семантикой указателя
```

File: _content/tour/rus/embedding.article [hunk 4]
```diff
@@ -82,7 +82,7 @@ can be promoted up to the outer type.
...
```
Comment (rumyantseva) [inline _content/tour/rus/embedding.article:85]:
```suggestion
        *user              // Встраивание с семантикой указателя
```

File: _content/tour/rus/embedding.article [hunk 5]
```diff
@@ -139,15 +138,15 @@ interface.
...
     Sending user email To john smith<john@yahoo.com>
 
-You can send the address of the admin value into the polymorphic function since
-embedding promotes the notify behavior up to the admin type.
+Вы можете передать адрес значения admin в полиморфную функцию, поскольку встраивание
+продвигает поведение notify вверх к типу admin.
 
     type admin struct {
-        *user  // Pointer Semantic Embedding
+        *user  // Встраивание семантики указателя
```
Comment (rumyantseva) [inline _content/tour/rus/embedding.article:149]:
```suggestion
        *user  // Встраивание с семантикой указателя
```

**Review by rumyantseva — COMMENTED** [review]
Looks great, just a few tiny suggestions.

File: _content/tour/eng/template/index.tmpl [hunk 6]
```diff
@@ -88,7 +88,6 @@
                     <option value="per">فارسی</option>
                     <option value="pol">Polski</option>
                     <option value="por">Português</option>
-                    <option value="tur">Türkçe</option>
```
Comment (alexshd) [inline _content/tour/eng/template/index.tmpl]: Duplication removed, order restored.

**Review by alexshd — APPROVED** [review]

File: _content/tour/rus/embedding.article (hunk 1 above)
Comment (alexshd) [reply _content/tour/rus/embedding.article:9]: So, I will add `embedding` as `Встраивание` and `promotion` as `продвижение`  to our dictionary.yml. I think later we need to scan the translation to see that all the translations in the dictionary are consistent in all the articles ...

Approving.

**Review by alexshd — APPROVED** [review]

**Pull request merged**